### ISSUE TITLE
Fix Nuget Readme rendering

### ISSUE THREAD

**Issue by nulltoken** [NCronJob-Dev/NCronJob#123]
**Describe the bug**
As seen on https://www.nuget.org/packages/NCronJob#readme-body-tab, the html trickery doesn't play well with nuger.org markdown rendering engine.

![image](https://github.com/user-attachments/assets/f68ac29b-6b24-4eb8-98ad-1cc843aa556a)

**Comment by linkdotnet** [comment]
Yeah we could have a separate `README.md` just for nuget - not sure why `<h1>` and stuff doesn't work.
The `img src` should probably use the github.com/NcronJob-Dev/... URL instead of a relative one.

**Comment by linkdotnet** [comment]
Basically we are blocked by: https://github.com/NuGet/NuGetGallery/issues/8644
